src/user/epics/index.js

```diff
@@ -2,25 +2,45 @@ import { combineEpics, ofType } from 'redux-observable';
 import { switchMap } from 'rxjs/operators';
 
 import * as actions from '../actions';
-import { userApi } from '../api';
-import { getCookie } from 'utils/cookie';
 
+// import { userApi } from '../api';
+
+// remove mocked one when backend will work properly on new microservices
```

Comment:
😄

Reply:
any problem ? :D better to have comments like that than not :P